### ISSUE TITLE
consult-eglot-symbols errors out

### ISSUE THREAD

**Issue by ville-h** [mohkale/consult-eglot#22]
Invoking `consult-eglot-symbols` errors out with: Symbol’s function definition is void: eglot-server-capable

This has happened [previously](https://github.com/mohkale/consult-eglot/issues/19).

**Comment by ville-h** [comment]
Are you able to observe the bug or do I need to provide further information?

**Comment by mohkale** [comment]
Sorry, been a bit swamped the last month. I'll take a look soon.

**Comment by mohkale** [comment]
Pushed a commit that should fix this. In future please notify through an issue but if I take a long time to respond I'd suggest adding a defalias in your init.el until it's fixed here. I've had to do that for other projects as well on occasion :disappointed:.

If the issue persists please let me know.

**Comment by ville-h** [comment]
Seems like consult has countered with a new move:
```
CONSULT ERROR: `consult-eglot-symbols' uses the old async API
convention and must be updated.  The `consult--async-*' API has been
updated in a backward-incompatible way.  For details, please see the
Consult CHANGELOG, the relevant git commit message and the docstring
of the `consult--async-pipeline' macro, which describes the updated
API.
```

**Comment by mohkale** [comment]
Yes, there's an open issue for this #24. I plan to look at it on the weekend. In the meantime might be worth downgrading to an older version of consult if you can.
